scripts/windows/build_log_inspector.bat

```diff
@@ -1,17 +1,24 @@
 @echo off
+setlocal
 
 echo Build Log Inspector
 echo.
 
-:: Set SDK_DIR as directory path
-for %%i in (%~dp0..\..) do SET SDK_DIR=%%~fi
-call %SDK_DIR%\scripts\lib\activate_python_venv.bat
+:: Set SDK_DIR to the repo root (two levels up from this script)
+for %%i in ("%~dp0..\..") do set "SDK_DIR=%%~fi"
```

Comment:
I have a bunch of fixes for wrapping the `~-dp0..\..` in double quotes, as well as the SET, though specifically this should be `set SDK_DIR="%%!fi"`.  Doing this should eliminate the need to double quote the "%SDK_DIR%" uses below.